modules/network/src/main/java/org/apache/ignite/internal/network/direct/stream/DirectByteBufferStreamImplV1.java

```diff
@@ -1365,13 +1365,22 @@ public IgniteUuid readIgniteUuid() {
     public <T> void writeMarshallable(T object, MessageWriter writer) {
         switch (marshallableState) {
             case 0:
+                writeBoolean(object == null);
```

Comment:
I just did it the same as for the UUID + it's actually smaller in size, because we don't write an empty collection of descriptors and an empty byte array.

Reply:
I see